ocs_ci/ocs/fdf_upgrade.py

```
@@ -0,0 +1,510 @@
+        """
+        Load FDF version-specific configuration file.
+
+        Args:
+            upgrade_version (str): FDF version to load config for
+
+        """
+        version_config_file = os.path.join(
+            constants.FDF_VERSION_CONF_DIR, f"fdf-{upgrade_version}.yaml"
+        )
+        if os.path.exists(version_config_file):
+            logger.info(f"Loading config file for FDF version: {upgrade_version}")
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)